ckan/public-midnight-blue/base/scss/_layout.scss

```diff
@@ -43,7 +43,6 @@
 
 .main .primary {
     border-radius: 20px;
-    background-color: $layoutPrimaryBackgroundColor;
```

Comment:
![image](https://github.com/user-attachments/assets/bfa3d9fd-95f5-47f0-8a1f-a95d9994b206)
This change disrupts the dataset page . Can we revert this please?